packages/utils/src/bitcoin/wallet.ts

```diff
@@ -0,0 +1,40 @@
+import { TypeOfAddress } from '@buildwithsygma/bitcoin';
```

Comment:
@wainola I think of `utils` package as providing access access to `composite` utils which combine functionality of two or more SDK sub packages, like `evm` and `substrate` for `hasEnoughLiquidity`. Can we move `bitcoin` specific utils to bitcoin package itself? and only keep utils that share logic across packages?

Reply:
solved and move it to the btc folder